presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/EnvironmentModule.java

```diff
@@ -19,6 +19,7 @@
 import io.prestosql.tests.product.launcher.env.common.Hadoop;
 import io.prestosql.tests.product.launcher.env.common.Kerberos;
 import io.prestosql.tests.product.launcher.env.common.Standard;
+import io.prestosql.tests.product.launcher.env.environment.SinglenodeKafka;
```

Comment:
> Migrate singlenode-kafka to PTL

I used this convention for commit messages:

"Convert singlenode-kafka to Launcher"